ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -11,11 +11,11 @@ import {
   ButtonVariant,
   IconName,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import { TextFieldSearch } from '../../component-library/text-field-search/deprecated';
 import { AccountListItem, CreateAccount, ImportAccount } from '..';
 import {
```

Comment:
Screenshots and screencasts showing no visual changes to UI

### Before

<img width="1512" alt="Screenshot 2023-12-10 at 3 39 20 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/43255501-4750-44a7-94cd-1c5893e96f5d">

### After

<img width="1512" alt="Screenshot 2023-12-10 at 3 39 33 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/f45c0266-d889-42ec-8df0-ecd4aef7fa2a">


https://github.com/MetaMask/metamask-extension/assets/8112138/8256451f-ed88-49fa-9987-c59b8e2855a2